openstack_controller/datadog_checks/openstack_controller/api.py

```diff
@@ -24,10 +26,19 @@ def create(logger, proxies, instance_config):
         paginated_limit = instance_config.get('paginated_limit')
         request_timeout = instance_config.get('request_timeout')
         user = instance_config.get("user")
+        openstack_sdk_config_file_path = instance_config.get("openstack_sdk_config_file_path")
+        openstack_sdk_cloud_name = instance_config.get("openstack_sdk_cloud_name")
+
+        api = None
+
+        if openstack_sdk_cloud_name is None:
```

Comment:
Let's add comments here to explain what is going on for people you don't have the full context.